embedded-hal/src/lib.rs

```diff
@@ -17,5 +17,5 @@ mod private {
 }
 
 // needed to prevent defmt macros from breaking, since they emit code that does `defmt::blahblah`.
-#[cfg(feature = "defmt-03")]
-use defmt_03 as defmt;
+#[cfg(feature = "defmt")]
+use defmt;
```

Comment:
this is not needed now that we're not renaming the dep anymore. (here and in other files either as `use defmt` or `use crate::defmt`)